hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -1244,8 +1244,8 @@ def describe_namespace(namespace_name)
     # Returns a list of namespaces in hbase
     def list_namespace(regex = '.*')
       pattern = java.util.regex.Pattern.compile(regex)
-      list = @admin.listNamespaceDescriptors.map(&:getName)
-      list.select { |s| pattern.match(s) }
+      list = @admin.listNamespaces
+      list.select {|s| pattern.match(s) }
```

Comment:
I think the warning pointing here `{|s|` which should be `{ |s|`

Reply:
Thank you. I was blind to that whitespace change.

Reply:
Actually I didn't change that line at all, so that was my confusion. But I will update it...